cmd/dockerd/options.go

```diff
@@ -27,8 +28,8 @@ var (
 	dockerTLSVerify = os.Getenv("DOCKER_TLS_VERIFY") != ""
 )
 
-// CommonOptions are options common to both the client and the daemon.
-type CommonOptions struct {
+// daemonFlags are options flags for the daemon (that were share with the client previously).
+type daemonFlags struct {
```

Comment:
This should probably be `daemonOptions`. They aren't actually flags. The flags are only ever stored on the Command itself. These are just the fields that receive values from flags, which we call options everywhere else.

We already have a `daemonOptions`, so maybe we just merge this struct with the existing `daemonOptions` ?

To reduce the size of the diff maybe just move `LogLevel`, and `Debug` to the existing `daemonOptions` struct, leaving host and tls fields, and this can be `hostOptions` or something like that?